docs/azure_arc_jumpstart/azure_arc_servers/oci/oci_terraform_ubuntu/_index.md

```diff
@@ -0,0 +1,198 @@
+* 
+    ```shell
+    source ./scripts/vars.sh
+    ```
+
+* Run the ```terraform init``` command which will download Terraform's AzureRM and OCI providers.
+
+    ![Screenshot of terraform init being run](./05.png)
+
+# Deployment
+
+* Run the ```terraform apply --auto-approve``` command and wait for the plan to finish. Upon completion, you will have an OCI virtual machine instance deployed and connected as a new Azure Arc enabled server inside a new Resource Group.
+
+* Open the Azure portal and navigate to the resource group "Arc-OCI-Demo". The virtual machine created on OCI will be visible as a resource.
+
+    ![Screenshot of Azure portal showing Azure Arc enabled server](./06.png)
+
+    ![Screenshot of OCI console showing OCI compute instance](./07.png)
+
+## Semi-Automated Deployment (Optional)
+
+As you may have noticed, the last step of the run is to register the VM as a new Azure Arc enabled server resource.
+
+![Screenshot showing azcmagent connect script](./08.png)
+
+If you want to demo/control the actual registration process, do the following:
+
+* In the [*install_arc_agent.sh.tmpl*](https://github.com/microsoft/azure_arc/blob/main/azure_arc_servers_jumpstart/oci/ubuntu/terraform/scripts/install_arc_agent.sh.tmpl) script template, comment out the "Run connect command" section and save the file.
+
+     ![Screenshot showing azcmagent connect script commented out](./09.png)
+
+* Get the public IP of the OCI instance by running ```terraform output```
+
+    ![Screenshot of terraform output being run](./10.png)
+
+* SSH the VM using the ```ssh -i my_oci_key ubuntu@x.x.x.x``` where x.x.x.x is the host ip.
+
+    ![Screenshot of SSH into OCI server](./11.png)
+
+* Export all the environment variables in [*vars.sh*](https://github.com/microsoft/azure_arc/blob/main/azure_arc_servers_jumpstart/oci/ubuntu/terraform/scripts/vars.sh)
+
+    ![Screenshot showing export of environment variables in var.sh](./12.png)
+
+* Run the following command
+
+    ```shell
+    sudo azcmagent connect --service-principal-id $TF_VAR_client_id --service-principal-secret $TF_VAR_client_secret --resource-group "Arc-OCI-Demo" --tenant-id $TF_VAR_tenant_id --location "westus2" --subscription-id $TF_VAR_subscription_id --tags "Project=jumpstart_azure_arc_servers"
+    ```
+
+    ![Screenshot of azcmagent connect being run](./13.png)
+
+* When complete, your VM will be registered with Azure Arc and visible in the resource group inside Azure Portal.
+
+## Delete the deployment
+
+* To delete all the resources you created as part of this demo use the ```terraform destroy --auto-approve``` command as shown below.
+
+  ![Screenshot showing terraform destroy being run](./14.png)
+
+* Alternatively, you can delete the OCI compute instance directly by terminating it from the [OCI Console](https://cloud.oracle.com/compute/instances). Note that it will take a few minutes for the instance to actually be removed.
```

Comment:
```suggestion
* Alternatively, you can delete the OCI Virtual Machine instance directly by terminating it from the [OCI Console](https://cloud.oracle.com/compute/instances). Note that it will take a few minutes for the instance to actually be removed.
```